test/models/docker_manifest_list_test.rb

```diff
@@ -8,7 +8,7 @@ class DockerManifestListTest < ActiveSupport::TestCase
 
     def setup
       FactoryBot.create(:smart_proxy, :default_smart_proxy)
-      @manifest_lists = YAML.load_file(MANIFESTS).values.map(&:deep_symbolize_keys)
+      @manifest_lists = Psych.unsafe_load_file(MANIFESTS).values.map(&:deep_symbolize_keys)
```

Comment:
This seems to be backwards incompatible so the tests are failing there.

Reply:
Damn so both versions are in use at some point? I guess I need to pin this.